src/messages/es.json

```diff
@@ -258,26 +352,24 @@
   "known_issues_result.empty": "Ningún resultado coincidente. Pruebe con otros filtros.",
   "known_issues_date.created": "Creado en",
   "known_issues_date.updated": "Actualizado en",
-  "tutorial_and_solutions_page.title": "Tutoriales y Soluciones",
-  "tutorial_and_solutions_page.description": "Accede a la documentación de todos los módulos de la plataforma y resuelve tus dudas.",
+  "tutorials_page.title": "Tutoriales hub",
+  "tutorials_page.description": "Navegue por nuestros tutoriales para mejorar su comprensión de la plataforma.",
   "date_text.created": "Publicado en",
   "date_text.updated": "Última actualización",
   "announcements_page_result.empty": "Ningún resultado coincidente.",
   "announcements_page_search.placeholder": "Buscar en Anuncios...",
   "search_result.empty": "Ningún resultado coincidente. Pruebe con otros filtros.",
   "faq_filter.title": "Produtos",
-  "faq_page.results_found": "resultados encontrados",
   "sitemap_page.title": "Mapa del sitio",
   "sitemap_page_section_additional_resources.title": "Recursos adicionales",
   "sitemap_page_section_additional_resources.support_rules": "Directrices de Soporte",
   "sitemap_page_section_additional_resources.known_issues": "Problemas Conocidos",
   "sitemap_page_section_additional_resources.faq": "Preguntas Frecuentes",
   "sitemap_page_section_additional_resources.announcements": "Anuncios",
-  "sitemap_page_section_additional_resources.dev_portal": "Portal de desarrolladores",
+  "sitemap_page_section_additional_resources.dev_portal": "Portal de Desarrolladores",
```

Comment:
why change this?

Reply:
This change is due to another modification occurring after the creation of this branch. I will restore the original information.